app/assets/stylesheets/components/_button.sass

```diff
@@ -0,0 +1,19 @@
+$Button-default-backgroundColor: Theme-color(button, default)
+$Button-text-color: Theme-color(text, lighter)
+$Button-height: 50px
+$Button-spacing: $Theme-spacing-base
+
+.Button
```

Comment:
Explain to me how does this component work?
There's the component `Button` and then there's a descendant `Button-default`?
Why?
What's the reason for having a descendant `Button-default`?